scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/context/TaskContextSerializer.java

```diff
@@ -59,6 +60,9 @@ public File serializeContext(TaskContext context, File directory) throws IOExcep
         try (ObjectOutputStream objectOutputStream = new ObjectOutputStream(new FileOutputStream(file))) {
             objectOutputStream.writeObject(context);
         }
+        if (context.isRunAsUser()) {
+            ForkerUtils.setSharedPermissions(file, false);
```

Comment:
could you create a separate method more expressive ? something like setSharedExecutablePermissions